guides/common/modules/ref_registration-methods.adoc

```diff
@@ -4,7 +4,7 @@
 You can use the following methods to register hosts to {Project}:
 
 Global registration::
-You generate a `curl` command from {Project} and run this command from an unlimited number of hosts to register them using provisioning templates over the {Project} API.
+You generate a `curl` or `wget` command from {Project} and run this command from an unlimited number of hosts to register them using provisioning templates over the {Project} API.
```

Comment:
```suggestion
You generate a registration command from {Project} and run this command on an unlimited number of hosts to register them by using provisioning templates over the {Project} API.
```
I'd use the generic term (fewer details) in this context. We elaborate on the command in the specific section.